src/AzureOpenAIProxy.Management/Database/Event.cs

```diff
@@ -4,7 +4,7 @@ public partial class Event
 {
     public string EventId { get; set; } = null!;
 
-    public string OwnerId { get; set; } = null!;
+    public string? OwnerId { get; set; }

```

Comment:
Is owner nullable in the database? I thought it wasn't

Reply:
> Is owner nullable in the database? I thought it wasn't

Yes, it is, I think you requested that ages ago, as it made it easier for you to create the event.

The models and db context were created using the EF scaffolding tool...

dg

Reply:
Ah no, it's not meant to be null, https://github.com/microsoft/azure-openai-service-proxy/commit/5ac3080a0e8060f7daba300bcb2cf12ce5f5b5d8 is the change and it's a mistake when I did the column type change.

It doesn't make sense to have an event that doesn't have an owner, if we had events without owners then how would we ever access them?

I'll create a separate bug to fix that.